### DISCUSSION TITLE
Working Days

### DISCUSSION THREAD

**Discussion by Napoleon-BlownApart** [davish/obsidian-full-calendar#579] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Enable a switch that differentiates weekdays and weekends, thus allowing a calendar entry that spans a number of days to only appear on working days, but not weekends (and holidays). 

### ✔️ Solution

Implement a switch that enables distinction between working days from non-working days.

**Comment by YouFoundJK** [comment]
@Napoleon-BlownApart Probably not quite what you ask for (there is no filter for hiding events), but you could use a combination of recurring events and/or Bussiness hours to set up all day recurring events only on weekdays and colorcode week days and working hours for ease of visibility, in my fork [here](https://github.com/YouFoundJK/plugin-full-calendar).

<img width="1560" height="974" alt="image" src="https://github.com/user-attachments/assets/1d630fb6-2b4c-4372-8f0d-bf7389f53312" />
